src/pages/Detail/Detail.js

```diff
@@ -1,7 +1,84 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import DetailTopImg from './DetailTopImg';
+import DetailMain from './DetailMain/DetailMain';
+import DetailAside from './DetailAside/DetailAside';
+import styled from 'styled-components';
 
 const Detail = () => {
-  return <div>Detail</div>;
+  const [detailData, setData] = useState({});
+  const params = useParams();
+  const { detailId } = params;
```

Comment:
useParams 받아오는 부분에서 구조분해할당하시면 될 것 같습니다!
```suggestion
  const { detailId } = useParams();
```